### DISCUSSION TITLE
Shouldn't ubuntu-latest be 19.10?

### DISCUSSION THREAD

**Discussion by pmatos** [community/community#25579] [Actions]
<p>It says in <a href="https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions" rel="nofollow noopener">https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions</a> that Ubuntu 18.04 can be reached through the <code>ubuntu-18.04</code> or <code>ubuntu-latest</code> tags but shouldn’t <code>ubuntu-latest</code> be 19.10?</p>

**Comment by electrum** [comment]
<p>18.04 and 16.04 are Long Term Support (LTS) versions, which are more appropriate for this use case as they receive security and other updates for much longer than other releases. Using non-LTS releases would require users to switch much more frequently. The next one will be 20.04. See <a href="https://ubuntu.com/about/release-cycle" rel="nofollow noopener">https://ubuntu.com/about/release-cycle</a></p>

**Comment by pmatos** [comment]
<p>Still 18.04 should not be ubuntu-latest. ubuntu-lts probably - since the latest one is 19.10.</p>

**Comment by welwood08** [comment, marked as answer]
<p>In this context, <code>latest</code> means “the most recent version provided by GitHub for use in workflows” and not “the most recent version released upstream”.</p>
